examples/delete_customer.go

```diff
@@ -1,20 +1,30 @@
 package main
 
 import (
+	"fmt"
+	"os"
+	"encoding/json"
 	"github.com/ncr4/tuneuptechnology-go"
 )
 
 func main() {
-	tuneuptechnology.DeleteCustomer(		
-		// Pass in your email and API key
-		&tuneuptechnology.Client{
-			Auth: "",
-			APIKey: "",
-		},
+	// Setup your email and API key
```

Comment:
Ditto on ENVs

Reply:
Consider also creating an initial 'credentials helper' file in the examples, to manage the API Email / Key, so prospective devs using your lib only have to set the ENVs / credentials in a single place to test all of your examples.

Reply:
The tests are intended as a "basic vanilla" approach to implementing the library. One-off script runs etc. There's definitely more interesting ways to handle it but I'd assume those are beyond the scope of showing the most basic way to get it running,

Reply:
Sure, but is testing out any of the examples currently, I would have to provide credentials separately for each individual example, vs having a single place to set them and having them all instantly work for me.
Certainly not a necessity, but an easy QOL improvement for people who want to test out the module quickly.

Reply:
I updated the docs to match passing env variables - I agree this would be nice but feel it's a bit out of the scope of this PR.